components/omega/configs/Default.yml

```diff
@@ -25,3 +25,60 @@ Omega:
     ViscDel2: 1.0e3
     VelHyperDiffTendencyEnable: true
     ViscDel4: 1.2e11
+  IOStreams:
+    InitialState:
+      UsePointerFile: false
+      Filename: OmegaMesh.nc
```

Comment:
Also, the initial decomposition (Decomp) can't use streams because parallel IO can't be set up until after Decomp. But I might still be able to at least read the mesh file name from the config.

Reply:
Okay, it sounds like having the mesh filename be read from a stream is silly because that would not be a stream you could modify (except for the filename).  So it sounds like the mesh filename should be a normal config option somewhere earlier in the yaml file.  It still would be convenient for Polaris if the mesh didn't always have to be named `OmegaMesh.nc` because it requires otherwise unnecessary logic specific to Omega.  (Obviously, this isn't a good name for the mesh/initial condition for MPAS-Ocean.)

Reply:
Not a problem - the important routines take the filename as an argument assuming we were going that route eventually. Early on, it was just easy to go with the hardwire-soft link route.